src/main/java/uk/gov/hmcts/reform/profilesync/service/impl/ProfileSyncServiceImpl.java

```diff
@@ -91,7 +93,6 @@ public String getS2sToken() {
             formParams.put("page", String.valueOf(counter));
             Response response  = idamClient.getUserFeed(bearerToken, formParams);
             ResponseEntity responseEntity = JsonFeignResponseHelper.toResponseEntity(response, new TypeReference<List<IdamClient.User>>() { });
-            Class clazz = response.status() > 300 ? ErrorResponse.class : IdamClient.User.class;
```

Comment:
Sonar: "Remove this useless assignment to local variable 'clazz' " - it was never used.
https://sonarcloud.io/project/issues?id=RDPROFILESYNCAPI&open=AWxuE_h2_Xcr_PhAhRjL&resolved=false